salt/proxy/junos.py

```diff
@@ -106,9 +106,18 @@ def init(opts):
             args[arg] = opts['proxy'][arg]
 
     thisproxy['conn'] = jnpr.junos.Device(**args)
-    thisproxy['conn'].open()
+    try:
+        thisproxy['conn'].open()
+    except Exception:
```

Comment:
Is there a more specific Exception that can be caught here rather than this broad one?